workshops/util.py

```diff
@@ -551,3 +551,32 @@ def update_event_attendance_from_tasks(event):
 
 def universal_date_format(date):
     return '{:%Y-%m-%d}'.format(date)
+
+
+def get_members(earliest, latest):
+    '''Get everyone who is a member of the Software Carpentry Foundation.'''
+
+    member_badge = Badge.objects.get(name='member')
+    instructor_badge = Badge.objects.get(name='instructor')
+    instructor_role = Role.objects.get(name='instructor')
+
+    # Everyone who is an explicit member.
+    explicit = Person.objects.filter(badges__in=[member_badge]).distinct()
+
+    # Everyone who qualifies by having taught recently.
+    implicit = Person.objects.filter(
+        task__role=instructor_role,
+        badges__in=[instructor_badge],
```

Comment:
On Tue, Nov 24, 2015 at 08:59:30AM -0800, Piotr Banaszkiewicz wrote:
> I didn't know about this "shortcut". However I don't think there's
> any significant advantage in going this way, so I'll merge without
> changing this line.

It's probably something that the SQL engine will optimize out, but for
dumb engines that don't I expect the singular comparison would be more
efficient.  Not a big deal for us, since we're not hitting any
performance bottlenecks that I know of.